osbenchmark/builder/executors/executor.py

```diff
@@ -1,17 +1,31 @@
-"""
-Executors are used to run shell commands on the cluster hosts. Implementations of this class will use various
-technologies to interface with the hosts of a cluster.
-"""
+import logging
+
+
 class Executor:
+    """
+    Executors are used to run shell commands on the cluster hosts. Implementations of this class will use various
+    technologies to interface with the hosts of a cluster.
+    """
     def __init__(self):
-        pass
+        self.logger = logging.getLogger(__name__)
```

Comment:
Yes it's just for the implementations. The idea of having it here was to avoid duplicating the same in the implementations that need it.

It seems to be pythonic to implement shared and inherited constructor logic in this way. I don't have a strong preference either way though so I'm fine moving it to the concrete implementations if that's the standard we want to follow.

Reply:
Looks like it does log as coming from the `executor` when instantiated here. Will remove this and instead instantiate in the implementations